### ISSUE TITLE
Can't accept follow request from umlaut domain

### ISSUE THREAD

**Issue by ben-utzer** [friendica/friendica#7764]
### Expected behavior
Following a person on mein.düsselblog.de should work. 

### Actual behavior
Following a person on mein.xn--dsselblog-q9a.de does not work, accepting the follow request from a person over there will end up on an empty page, on "/dfrn_confirm". 

### Steps to reproduce the problem
Find a user on mein.xn--dsselblog-q9a.de and try to follow that person. 

### Friendica version you encountered the problem
2019.09

### Friendica source (git, zip)
git

### PHP version
PHP 7.3.10

### SQL version
10.4.8-MariaDB

**Comment by MrPetovan** [comment]
Is it a Friendica server?

**Comment by ben-utzer** [comment]
> Is it a Friendica server?
yes, but Version 2018.05

**Comment by MrPetovan** [comment]
This could prove problematic if the problem comes from the receiving server and they don't update.

**Comment by ben-utzer** [comment]
> This could prove problematic if the problem comes from the receiving server and they don't update.
we will see, they will update, the installation is rather new I thought, but I just saw it is that version. 

However, maybe the following worked, but just the confirmation page is not loaded on my end. Unfortunately in the mean time I did follow that person from my side and now I saw in the contact that the account is sharing with me and that I received updates. 
Now I am not sure why the contact request ist still there to accept.
